src/modules/ekf2/EKF/common.h

```diff
@@ -101,7 +101,8 @@ enum MagFuseType : uint8_t {
 	// Integer definitions for mag_fusion_type
 	AUTO    = 0,   	///< The selection of either heading or 3D magnetometer fusion will be automatic
 	HEADING = 1,   	///< Simple yaw angle fusion will always be used. This is less accurate, but less affected by earth field distortions. It should not be used for pitch angles outside the range from -60 to +60 deg
-	NONE    = 5    	///< Do not use magnetometer under any circumstance..
+	NONE    = 5,   	///< Do not use magnetometer under any circumstance.
+	INIT    = 6     ///< Use the mag for heading initialization only.
```

Comment:
You don't want to fill the gaps?

```suggestion
	INIT    = 3     ///< Use the mag for heading initialization only.
```

Reply:
Yes, but I already had it implemented as 6 in an older version of the code where we still had the other mag types... Keeping it at 6 would make the upgrade easier for the current users of that older version.

Reply:
Otherwise I could make a param translation to convert a type 6 to 3.